### ISSUE TITLE
Can't send or receive messages from iOS, receive hundreds of "out of sync"-messages and other problems

### ISSUE THREAD

**Issue by St3fi** [signalapp/Signal-iOS#5157]
- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
- I cannot send or receive messages on iPhone
- I receive hundreds of notifications that turn out to be empty
- I cannot find out the sender of the messages since nothing happens when I click on the notifications (except that Signal opens)
- Resetting the conversation is not possible
- Signal on iOS is very slow and often crashed, Phone is running hot, battery is drained
- Desktop application synchronized >1500 messages on startup after bug first occurred

### Steps to reproduce
- (re-)install signal
- link to a desktop application
- wait for 1 day

**Actual result:** Describe here what happens after you run the steps above (i.e. the buggy behaviour)
- I got hundreds of empty messages
- I cannot sent or receive messages on the phone

**Expected result:** Describe here what should happen after you run the steps above (i.e. what would be the correct behaviour)
- I can send and receive messages
- I do not get empty messages
- When I click on the Notification, something should open
- Messages should be synced between desktop-application and phone
- I should be able to reset a conversation somehow

### Device info

**Device**: iPhone X

**iOS version**: 15.0.1

**Signal version:** 5.21.2.4

### Link to debug log

**iOS debug log**:
Log URL: https://debuglogs.org/9544c802695246cc460df64eea4590ee1aa9e8003e1a1d11005cb34187b63ecc.zip
Locale: en_DE

**Desktop app debug log**:
https://debuglogs.org/bbe7aa4f751d520aec705262cbaa874cd7c3aaa9a03444153a854ce7efac5f22.gz

![IMG_4584_smol](https://user-images.githubusercontent.com/19670470/135817635-735279df-621b-486a-84df-3622753b32c7.PNG)

![IMG_4583](https://user-images.githubusercontent.com/19670470/135818370-f0ff3c8b-92d7-4c90-a8c7-290771c20631.PNG)

**Comment by WY9888** [comment]
Can't send or receive messages from iOS, receive hundreds of "out of sync"-messages and other problems

**Comment by St3fi** [comment]
Hi,

i had to reinstall Signal multiple times on my mobile phone to make it work again, since not using Signal was no option. (However, my whole conversations are lost since there is no option to back them up.)

Now, the error did not occur so far. I guess if it would, I'd have to reinstall Signal again. So the last logs that I retrieved from when the error first occurred (which I posted above) are all that I can provide.

Reinstalling Signal is a very drastic fix ... but I don't know what can be done to further investigate this bug. Is there anything that I can do?
